javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,79 @@
+console.log("hello");
+//building a voice assistant.
+//function called getReply(command).
+let personName = [];
+const todoList = [];
+let today = new Date();
+let date = today.getDate();
+
+
+let options = { year: 'numeric', month: 'long' };
+const formatedDate = date + ". of " + today.toLocaleString('en-US', options);
+
+let operators = [];
+
+function getReply(command) {
+    if (command == " " || typeof command !== "string") {
```

Comment:
What about `getReply("");` ? 🙂 
Hint: you can also add else at the end of the function
Be careful with `==` and `===`. Which one you should use? Good reading https://www.freecodecamp.org/news/javascript-triple-equals-sign-vs-double-equals-sign-comparison-operators-explained-with-examples/

Reply:
will change it.